charts/rancher-vsphere-cpi/rancher-vsphere-cpi/100.0.0/templates/vsphere-cpi-rbac.yaml

```diff
@@ -105,23 +106,24 @@ roleRef:
 subjects:
 - apiGroup: ""
   kind: ServiceAccount
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
   namespace: {{ .Release.Namespace }}
 - apiGroup: ""
   kind: User
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
 subjects:
 - kind: ServiceAccount
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
   namespace: {{ .Release.Namespace }}
 - kind: User
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
```

Comment:
Where is the `User` CR from? I do not find its definition in the chart.

Reply:
they are defined in another chart: `rancher-vsphere-csi`

Reply:
resolved [here](https://github.com/rancher/charts/pull/1220#discussion_r661031973)